NMS-14537: Add option to disable IO collection (#5042)
OpenNMS/opennms: docs/modules/operation/pages/device-config-backup/configuration.adoc

config:update
----

== SSH Session I/O Collection

> By default, the ssh scripting service collects all ssh session output for debugging purposes and saves it in the poller's status. 

Author: It's stored in the poller's status so the karaf shell has access to it and can print it if needed. But everything that has access to the poller's status has access to it as well.

> By default, the ssh scripting service collects all ssh session output for debugging purposes and saves it in the poller's status. 

Author: I changed 'saves' to 'stores', as 'save' could imply persisting in a database.

> To disable this, you can configure the scripting service on both Horizon and Minion as below:

Author: You need to disable it on all minions, updated.

                Whether to persist config or not
----

> NOTE: Above karaf commands won't persist config by default, use -p option to persist config in {page-component-title} Database.

Reviewer: ```suggestion
NOTE: The previous Karaf commands will not persist configuration by default.
Use the -p option to persist configuration in the {page-component-title} database.
```
